blog/app/controllers/followers_controller.rb

```diff
@@ -0,0 +1,19 @@
+ class FollowersController < ApplicationController
+  before_action :load_user
+
+  def index
+    @title = t "users.controllers.follower"
```

Comment:
cái này I18n thì hoàn toàn có thể lấy trực tiếp được mà, cần gì viết ở đây chứ ?

Reply:
loi cua em a :>